bin/selection_utils.py

```diff
@@ -161,58 +161,98 @@ def target_distance_fcn(args, this_seq, target_seqs):
     return itarget, tdist
 
 # ----------------------------------------------------------------------------------------
+def count_aa(aa_seq,which_aa):
+    list_of_aa = list(aa_seq)
+    res1 = []
+    for i in range(len(list_of_aa)):
+        if list_of_aa[i] == which_aa:
+            res1.append(i)
+    return len(res1)
+
+def count_not_that_aa(aa_seq,which_aa):
```

Comment:
it may not make a difference, but i think something like `return len([a for a in aa_seq if a!=which_aa])` would be faster here (also maybe clearer?). Also just two general things -- it's best to have at least a few word description for even small functions (so you don't have to read the whole function to figure out what's getting counted where), and could you add spaces after commas to match the style elsewhere?

Reply:
You are right- that is a better  function. And yes, I can add spaces.